core/kazoo_ledgers-1.0.0/src/kazoo_ledger.erl

```diff
@@ -27,8 +41,9 @@
 %%
 %% @end
 %%--------------------------------------------------------------------
--spec new() -> ledger().
-new() -> #kz_ledger{}.
+-spec new() -> wh_json:object().
+new() ->
+    wh_json:from_list([]).
```

Comment:
I wish there was a function in `wh_json` that created a new JSON object...we could call it `wh_json:new/0` or something...